src/kiwixchoicebox.cpp

```diff
@@ -171,28 +171,17 @@ void KiwixChoiceBox::setSelections(QStringList selections, QString defaultSelect
     setSelections(sList, defaultSelection);
 }
 
-void KiwixChoiceBox::setSelections(SelectionList selections, QString defaultSelection)
+void KiwixChoiceBox::setSelections(SelectionList selections, QStringList defaultSelection)
 {
     choiceSelector->clear();
     for (const auto &selection: selections)
     {
         auto item = new KListWidgetItem(beautifyString(selection.second));
         item->setData(Qt::UserRole, selection.first);
         choiceSelector->addItem(item);
-        if (selection.second ==  defaultSelection)
-        {
-            m_defaultItem = item;
```

Comment:
`m_defaultItem` is not changed anymore.
Either we remove it from the class or we have to handle it somehow.